### ISSUE TITLE
Resnet34 with pretrained weights on TensorFlow 2 

### ISSUE THREAD

**Issue by ectg** [qubvel/classification_models#53]
Hi,

Thanks for your work. I would like to train a resnet34 model starting with pretrained imagenet weights. This repo works fine on TF1, but not on TF2. Has this code been ported to TensorFlow 2?

Thanks.

**Comment by adeeb10abbas** [comment]
Yes, it has. It's working fine for me. I am using Resnet18. What's not working for you?
